iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/RetrieveRemoteResourcesCheck.java

```diff
@@ -40,9 +41,19 @@ public class RetrieveRemoteResourcesCheck implements IacCheck {
 
   private static final String MESSAGE = "Replace this invocation of %s with the ADD instruction.";
 
+  private static final List<String> WGET_AUTH_FLAGS = List.of("--http-user", "--http-password", "--proxy-user", "--proxy-password", "--load-cookies");
   private static final Predicate<String> WGET_DOWNLOAD_FLAG_PREDICATE = startsWithIgnoreQuotes("-O", "--output-document");
   private static final Predicate<String> URL_PREDICATE = startsWithIgnoreQuotes("http");
 
+  // By setting header manually: -H "Authorization ...", --header "Authorization ..." , -H "X-Auth-Token ..." , --header "X-Auth-Token ...".
```

Comment:
Not sure what is the purpose of this comment here. The list of flags below doesn't seems to be related to them.